content/luau/Concepts/Operators/operators.md

```diff
@@ -0,0 +1,150 @@
+---
+Title: 'Operators'
+Description: 'Operators are built-in functions that perform broadly logical or mathematical computations.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Operators'
+  - 'Math'
+  - 'Numbers'
+  - 'Logical'
+  - 'Arithmetic'
+CatalogContent:
+  - 'paths/computer-science'
+  - 'learn-lua'
+  - 'introduction-to-game-development'
+---
+
+**Operators** are built-in functions that perform broadly logical or mathematical computations. An operator takes in one or more arguments, performs some computations based on those arguments, and returns a value.
+
+## Logical Operators
+
+Logical operators mirror the behavior of logical concepts like conjunction, disjunction, and negation. Because Luau is not a strongly-typed language, logical operators in Luau will accept arguments other than `true` and `false`. Note also that Luau treats a value of `nil` as equivalent to `false` in the context of a logical operator.
+
+| Logical Operator |                                          Behavior                                                      |
+| :--------------: | :----------------------------------------------------------------------------------------------------: |
+|      `and`       |   Returns the first argument if the first argument is `false`; otherwise returns the second argument   |
+|      `or`        | Returns the first argument if the first argument is not `false`; otherwise returns the second argument |
+|      `not`       |           Returns `true` if and only if the argument is `false`; otherwise returns `false`             |
+
+### Examples
+
+```pseudo
+print(true and false) -- false
+print(false and false) -- false
+print(true and true) -- true
+print(1 and 2) -- 2
+print("Hello" and "World") -- World
+print(true or false) -- true
+print(false or false) -- false
+print(true or true) -- true
+print(1 or 2) -- 1
+print("Hello" or "World) -- Hello
+print(not false) -- true
+print(not nil) -- true
+print(not true) -- false
+print(not 1) -- false
+print(not "Hello World") -- false
+```
+
+## Mathematical Operators
+
+Mathematical operators include arithmetic and relational (comparative) operators.
+
+### Relational Operators
+
+Relational operators compare two arguments numerically (greater than, equal to, etc.) and return a boolean value of `true` or `false`.
+
+| Relational Operator|                                                       Behavior                                                 |
+| :----------------: | :------------------------------------------------------------------------------------------------------------: |
+|         ==         |                    Returns `true` if the two arguments are equal; otherwise returns `false`                    |
```

Comment:
```suggestion
|         `==`        |                    Returns `true` if the two arguments are equal; otherwise returns `false`                    |
```

Reply:
update the other rows as above